### ISSUE TITLE
Superior location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11559]
Preferred term label:
Superior location

Synonyms

    Upper location

    Superiorly located

    Cranial location

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to a position above or higher than another structure. In ophthalmology, this is frequently used to describe findings such as superior visual field defects, superior retinal detachments, or superior corneal pannus. PMID:22131396

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Trachoma (Orphanet:3334) (e.g., superior corneal pannus)

    Glaucoma (OMIM:137760) (e.g., superior arcuate scotoma)

Your nano-attribution (ORCID)
0000-0001-9042-0508
